compiler/modules/nback/scaffold.bal

```diff
@@ -235,16 +243,15 @@ class Scaffold {
         return d:location(self.file, pos);
     }
 
-    function setDebugLocation(llvm:Builder builder, bir:Position pos, "file"? fileOnly = ()) {
+    function setDebugLocation(llvm:Builder builder, bir:Position pos, DebugLocationTarget target) {
         DISubprogram? diFunc = self.diFunc;
         if diFunc != () {
             ModuleDI di = <ModuleDI>self.mod.di;
             DILocation loc;
-            if fileOnly == () {
-                var [line, column] = self.file.lineColumn(pos);
-                loc = di.builder.createDebugLocation(self.mod.llContext, line, column, self.diFunc);
+            if target > CALL_INSTRUCTION && !di.debugFull {
```

Comment:
I am sure we can do better than this...

Reply:
changed such that in full debug all locations will have the position (this have no effect on the panic error message)